data/hai/unfettered side missions.txt

```diff
@@ -0,0 +1,272 @@
+# Copyright (c) 2014 by Michael Zahniser
```

Comment:
I think it would be 2023, your name/username.

Reply:
I would but there is  an MZ mission there too

Reply:
Large changes mean that you can change the name, I think?

Reply:
```suggestion
# Copyright (c) 2014 by Michael Zahniser for the first mission
# Copyright (c) 2023 by Hurleveur
```

Reply:
That works.